### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Help" button in  Run on web app containers window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1441]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Tab to Help (?) button and hit Space bar and observe keyboard focus .

**Note:** Same issue is observed in create web app in Run on web app containers window.

**Expected Result:**
Keyboard focus is should visible on "Help" button in Run on web app containers window.

**Actual Result:**
Keyboard focus is not visible on "Help" button in Run on web app containers window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![focus not visible on help button in run on web app containers window](https://user-images.githubusercontent.com/22956130/35728599-f81f839e-07c0-11e8-8310-50f33aeb756d.png)

**Comment by Eskibear** [comment]
https://github.com/Microsoft/azure-tools-for-java/issues/1456

**Comment by WiproPSAccTest** [comment]
Verified in 2017.3.5 Community.
Issue still exists

![image](https://user-images.githubusercontent.com/22956130/39819265-a358d5f8-53c0-11e8-93ad-a365764b8583.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749899
